ksql-common/src/test/java/io/confluent/ksql/schema/ksql/types/SqlMapTest.java

```diff
@@ -48,7 +48,7 @@ public void shouldReturnValueType() {
   @Test
   public void shouldImplementToString() {
     assertThat(SqlMap.of(SOME_TYPE).toString(), is(
-        "MAP<VARCHAR, "
+        "MAP<STRING, "
```

Comment:
As mentioned in the PR description, where previously some code used `VARCHAR` and some `STRING`, the PR sees the code base starting to standardize on one way, which is controlled by `SqlType`.